### DISCUSSION TITLE
can I transition a production app from the official SDKs?

### DISCUSSION THREAD

**Discussion by ddnn55** [GitLiveApp/firebase-kotlin-sdk#131] [Q&A]
First, this is awesome! Hoping to give it a try soon.

Question: I have an app in production that uses the official SDKs on iOS and Android. How hard would it be to start using `firebase-kotlin-sdk`, and not break any of my user's persistent authentication and so on? It sounds like `firebase-kotlin-sdk` is just a wrapper around the official SDKs, so presumably this should work, but just want to double check.

**Comment by nbransby** [comment]
yes that will work fine
